base/Dockerfile.fedora

```diff
@@ -1,4 +1,4 @@
-FROM fedora:26
+FROM registry.fedoraproject.org/fedora:27
```

Comment:
This shouldn't be needed as this the [default configured](https://github.com/lcarva/osbs-box/blob/c64ad3d2e651213653b5fbc72df15565eb8674b3/shared-data/opt/local/osbs/osbs.conf#L23) `source_registry_uri`

Reply:
This is the base image used to build the other services, if I do a docker pull fedora it will default to docker.io.

Reply:
You are, of course, correct :)